sos/plugins/manila.py

```diff
@@ -14,31 +14,28 @@
 # along with this program; if not, write to the Free Software
```

Comment:
The patch description and contents are completely different - why not just squash this into the previous commit (nobody cares that 'cmds' was renamed to 'nas' as you were working on it..), and then write a single commit message that accurately describes the whole commit?

It is much less confusing for reviewers, and anyone who has to try to make sense of the history in the future.